### PR TITLE
chore: setup release 1.0.0

### PR DESCRIPTION
**Pull request by skuruppu**
Release-As: 1.0.0

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -14,9 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-[egg_info]
-tag_build = dev
-
```
Comment (skuruppu) [inline setup.cfg:19]: @IlyaFaer what is this for and is it ok that I removed it? I don't want this to be marked as a dev build (or for the versions to have any suffix).

Comment (IlyaFaer) [reply setup.cfg:19]: Yeah, it can be deleted

**Review by IlyaFaer — APPROVED** [review]
I suppose now we can erase these lines from the readme:
Безымянный
Otherwise LGTM!

**Comment by skuruppu** [comment]
> I suppose now we can erase these lines from the readme: Безымянный Otherwise LGTM!

Great catch @IlyaFaer. Thanks!

**Review by larkee — APPROVED** [review]

**Pull request merged**